_inc/client/traffic/verification-services.jsx

```diff
@@ -16,11 +16,30 @@ import {
 import { ModuleSettingsForm as moduleSettingsForm } from 'components/module-settings/module-settings-form';
 import SettingsCard from 'components/settings-card';
 import SettingsGroup from 'components/settings-group';
+import JetpackBanner from 'components/jetpack-banner';
 
 export const VerificationServices = moduleSettingsForm(
 	React.createClass( {
+		activateVerificationTools: function() {
+			return this.props.updateOptions( { [ 'verification-tools' ]: true } );
```

Comment:
I'm not sure, I was following the pattern of other updateOptions usage from when you refactored them